taosbenchmark-user-manual-CN.md

```diff
@@ -424,61 +277,80 @@ taosBenchmark 是TDengine进行性能测试的工具应用程序，可以进行T
+| | filetype | 文件类型，指定哪种测试，对于订阅测试，需要
+| |cfgdir | taos配置文件的目录。
+| | host | taosd服务器的FQDN，默认为localhost。
+| | port | taosd服务器的端口号，默认为6030。
+| |user | 连接taosd服务器的用户名，默认为root。
+| |password | 连接taosd服务器的密码，默认为taosdata。
+| |databases | 数据库名称，需要             
+| |confirm_parameter_prompt | 在执行过程中是否通过确认提示。
+| specified_table_query/super_table_query | concurrent/threads | 执行sql的线程数，默认为1。
+| specified_table_query/super_table_query | interval | 执行订阅的时间间隔，默认为0。
+| specified_table_query/super_table_query | restart | no: 继续之前的订阅，yes: 开始新的订阅。
+| specified_table_query/super_table_query | keepProgress | 是否保留订阅的进度。
+| specified_table_query/super_table_query | resubAfterConsume | 是否取消订阅，然后再次订阅？
+| super_table_query | stblname | supertable的名称，必须的。
+| sqls | [sql](#sql) | sql命令，必填
+| sqls | result | 查询结果的结果文件，没有则为空。
 
-"user": 用户名。可选项，缺省是“root“。
+- #### -i/--插值间隔
 
-"password": 密码。可选项，缺省是“taosdata"。
+  只有当隔行扫描(-B/-interlace-rows)大于0时才起作用。该
+  意味着线程在为每个子表插入隔行扫描记录后，会在该时间段内睡觉。
+  隔行扫描的时间。
 
-"databases": 数据库名称。必选项。**
+- #### -B/--interlace-rows
 
-"confirm_parameter_prompt": 执行过程中提示是否确认，为no时，执行过程无需手工输入enter。可选项，缺省是no。
+  如果它的值为0，意味着逐步插入，线程将逐个插入到
+  子表。如果它的值大于零，线程将首先插入
+  到第一个子表的行数，然后是第二个、第三个，以此类推。
+  以此类推，当所有的子表都被插入了该行数后，线程将从第一个子表重新开始插入。
+  将从第一个子表重新开始插入，依次进行。
 
-"specified_table_query": 指定表的订阅。
+- #### -l/--columns
 
-"concurrent": 并发执行sqls的线程数，可选项，缺省是1。每个线程都执行所有的sqls。
+  如果同时设置了-l/--columns和-b/--data-type，它将检查列数是否达到
+      列号是否达到-b/--数据类型的-l/--columns，如果是的话。
+      忽略这个选项，或者它将继续增加列数以达到这个数字
+      所有的int数据类型。
 
-"mode": 订阅模式。目前支持同步和异步订阅，缺省是sync。
+- #### -A/--tag-type
 
-"interval": 执行订阅的间隔，单位是秒。可选项，缺省是0。
+  设置超级表的标签类型，nchar和binary也可以设置长度，例如
+  如
 
-"restart": 订阅重启。"yes"：如果订阅已经存在，重新开始，"no": 继续之前的订阅。(请注意执行用户需要对 dataDir 目录有读写权限)
+  ```
+  taosBenchmark -A INT,DOUBLE,NCHAR,BINARY(16)
+  ```
 
-"keepProgress": 保留订阅信息进度。yes表示保留订阅信息，no表示不保留。该值为yes，restart为no时，才能继续之前的订阅。
+  默认是INT,BINARY(16)。
 
-"resubAfterConsume": 配合 keepProgress 使用，在订阅消费了相应次数后调用 unsubscribe 取消订阅并再次订阅。
+- #### -b/--数据类型
 
-"sql": 查询语句。必选项。
+  与-A/--tag-type相同，但用于列，默认为FLOAT,INT,FLOAT
 
-"result": 查询结果写入的文件名。可选项，缺省是空，表示查询结果不写入文件。
+- #### -O/--紊乱度
 
-"super_table_query": 对超级表中所有子表的订阅。
+  乱序时间戳的比例，最大为50
 
-"stblname": 超级表名称。必选项。
+- #### -R/--失序范围
 
-"threads": 并发执行sqls的线程数，可选项，缺省是1。每个线程都执行所有的sqls。
+  只有当-O/--disorder大于0时才有效，并且disorder意味着
+  时间戳将在该范围内减少随机数毫秒。
+  生成。
 
-"mode": 订阅模式。
+- #### prepared_rand
 
-"interval": 执行sqls的间隔，单位是秒。可选项，缺省是0。
+  作为数据源预先生成的随机数据的数量，小的prepare_rand
+  会节省内存，但会减少数据种类。
 
-"restart": 订阅重启。"yes"：如果订阅已经存在，重新开始，"no": 继续之前的订阅。
+- #### sql
 
-"keepProgress": 保留订阅信息进度。yes表示保留订阅信息，no表示不保留。该值为yes，restart为no时，才能继续之前的订阅。
-
-"resubAfterConsume": 配合 keepProgress 使用，在订阅消费了相应次数后调用 unsubscribe 取消订阅并再次订阅。
-
-"sql": " select count(*) from xxxx "。查询语句，其中表名必须写成 “xxxx”，实例会自动替换成子表名。
-
- "result": 查询结果写入的文件名。可选项，缺省是空，表示查询结果不写入文件。 注意：每条sql语句后的保存结果的文件不能重名，且生成结果文件时，文件名会附加线程号。
-```
+  对于超级表的查询sql，在sql命令中保留 "xxxx"，程序会自动将其替换为超级表的所有子表名。
```

Comment:
according to recent document standards, we should use 'SQL' in document instead of 'sql'.